corehq/apps/reports/standard/deployments.py

```diff
@@ -165,13 +168,20 @@ def selected_app_id(self):
         return self.request_params.get(SelectApplicationFilter.slug, None)
 
     @quickcache(['app_id'], timeout=60 * 60)
-    def get_app_name(self, app_id):
+    def _get_app_details(self, app_id):
```

Comment:
another option here is to use the `application_brief` view which is a much lighter version of the app and already includes all the details you want. I think you could probably cache the whole thing. One issue though is that the view only has Apps and not Builds.

Reply:
I am kind of sure that this would actually be a build each time so i was thinking of using `get_app_cached` instead. What do you think about that?

Reply:
I guess you could. I still think caching the whole app is unnecessary

Reply:
Agreed. Not caching that app anymore but we are still calling `get_app` which can be further improved by using `get_app_cached`.